Formula/coin3d-coin.rb

```diff
@@ -0,0 +1,55 @@
+class Coin3dCoin < Formula
+  desc "OpenGL-based Open Inventor 2.1 API implementation"
+  homepage "https://coin3d.github.io/"
+  url "https://github.com/coin3d/coin/archive/Coin-4.0.0.tar.gz"
+  sha256 "b00d2a8e9d962397cf9bf0d9baa81bcecfbd16eef675a98c792f5cf49eb6e805"
+  license "BSD-3-Clause"
+  head "https://github.com/coin3d/coin.git"
+
+  livecheck do
+    url :stable
+    regex(/^Coin-(\d+\.\d+\.\d+)$/i)
+  end
+
+  depends_on "cmake" => :build
+  depends_on "doxygen" => :build
+  depends_on "ninja" => :build
+  depends_on "boost"
+
+  def install
+    args = std_cmake_args + %w[
+      -GNinja
+      -DCMAKE_BUILD_TYPE=Release
+      -DCOIN_BUILD_MAC_FRAMEWORK=OFF
+      -DCOIN_BUILD_DOCUMENTATION=ON
+      -DOIN_USE_CPACK=OFF
```

Comment:
```suggestion
      -DCOIN_USE_CPACK=OFF
```
Is the missing `C` intentional?

Reply:
Ack. I'll remove that in the update formula.

Reply:
Great; hopefully you won't need to fake an empty `CMakeLists.txt` with the missing `C` in place